src/nav.config.ts

```diff
@@ -194,6 +194,10 @@ const fullNav: FullNav = {
           title: 'Using GitHub Actions',
           href: '/guides/getting-started/github-actions',
         },
+        {
+          title: 'Using Azure Devops',
```

Comment:
Should we title this as Azure Pipelines instead? Just a thought

Reply:
Sure sounds good!